cmd/host/main.go

```diff
@@ -51,6 +53,10 @@ func main() {
 						Name: "gateway",
 						IP:   net.ParseIP("192.168.127.1"),
 					},
+					{
+						Name: "host",
+						IP:   net.ParseIP(virtualHostIP),
+					},
```

Comment:
Why do we want to be able to use `host.api.testing` to reach the host though?

Reply:
Sometimes when developing the user might want to reach services running on the host. 
When using traditional networking, the user take a good known IP and it works. I guess 192.168.130.1 for crc+libvirt. 
With vsock, we need to create a virtual IP. Having a DNS name for that, it's just more convenient.

Reply:
I guess this will need documentation at some point, as people won't find it by themselves. Can you add this rationale to the commit log, and maybe explicitly mention the virtual host IP in addition to the hostname?

Edit: I see just now there was a sample in the PR description, which shows  how much attention I am paying to these ^^